packages/avatar/src/avatar.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import { Avatar } from './index';
+import { render } from '@react-email/render';
+
+describe('render', () => {
+  beforeEach(() => {
+    jest.restoreAllMocks();
+    jest.resetModules();
+  });
+
+  it('renders the <Avatar> component', () => {
+    const actualOutput = render(<Avatar src="cat.jpg" name=" My Cat" />);
```

Comment:
Can we also use a real source? Maybe from https://plaiceholder.co/
```suggestion
    const actualOutput = render(<Avatar src="cat.jpg" name="My Cat" />);
```